pkg/local_object_storage/pilorama/boltdb.go

```diff
@@ -627,6 +627,17 @@ func (t *boltForest) TreeGetOpLog(cid cidSDK.ID, treeID string, height uint64) (
 // TreeDrop implements the pilorama.Forest interface.
 func (t *boltForest) TreeDrop(cid cidSDK.ID, treeID string) error {
 	return t.db.Batch(func(tx *bbolt.Tx) error {
+		if treeID == "" {
+			c := tx.Cursor()
+			prefix := bucketName(cid, treeID)
```

Comment:
do we need a `bucketName` call here? is `[]byte(cid.EncodeToString())` enough and more verbose?

Reply:
Yes, looks better, added.